Adding storage gloss terms (#335)
redhat-documentation/supplementary-style-guide: supplementary_style_guide/glossary_terms_conventions/general_conventions/m.adoc

*See also*: xref:messaging-activemq-management[Messaging - ActiveMQ], xref:messaging-subsystem[messaging subsystem]

[[metadata]]
==== image:images/yes.png[yes] metadata (noun)

> *Description*: Generically, data that provides information about one or more other pieces of data.

Reviewer: ```suggestion
*Description*: _Metadata_ is a generic term for data that provides information about one or more other pieces of data.
```

*See also*: xref:windows-server[Windows Server]

[[migrate]]
==== image:images/caution.png[with caution] migrate (verb)

> *Description*:  To _migrate_ data is to move data automatically from one location to another, usually to proactively or reactively improve cluster performance. OpenShift Container Platform uses the phrases _auto-migrate_ or _automatically migrate_ in this use case.

Reviewer 2: "Migration" can refer to a lot of things and concepts, so the first sentence needs to be updated to specify the product or situation it applies to.

In virtualization, _live migration_ refers specifically to a virtual machine being moved from one host to another while remaining operational.

> *Use it*: with caution - for clarity, use the full term first before shortening to _migrate_.

Reviewer 2: Add the part about using the full term first before shortening to the definition instead.

Reviewer 3: I'm going to yank this entry. It seems like the other terms might be actually what should be brought in as entries. CC @lpettyjo @laubai if you feel strongly that these terms should be added, let us know if we should add them in a followup PR

*See also*: xref:windows-server[Windows Server]

[[migrate]]
==== image:images/caution.png[with caution] migrate (verb)

> *Description*: To automatically move data from one location to another, usually to proactively or reactively improve cluster performance. OpenShift Container Platform uses "auto-migrate" or "automatically migrate" in this case.

Reviewer: ```suggestion
*Description*: To _migrate_ data is to move data automatically from one location to another, usually to proactively or reactively improve cluster performance. OpenShift Container Platform uses the phrases _auto-migrate_ or _automatically migrate_ in this use case.
```

> In virtualization, "live migration" refers specifically to a virtual machine being moved from one host to another while remaining operational.

Reviewer: ```suggestion
In virtualization, _live migration_ refers specifically to a virtual machine being moved from one host to another while remaining operational.
```

*See also*:

[[multipath]]
==== image:images/caution.png[with caution] multipath (noun)
*Description*: Device Mapper _Multipath_ Input Output (DM-MPIO), also known as DM _Multipathing_ or _multipath_, is a Linux utility that enables multiple input/output paths between server nodes and storage arrays to be configured into a single device path to provide redundancy and improve performance.

> *Use it*: with caution - this term may be unclear on its own.

Reviewer 2: ```suggestion
*Use it*: with caution
```
